src/contract.js

```diff
@@ -1,19 +1,32 @@
+import transfer from "./functions/transfer.js";
```

Comment:
Will imports work with esbuild's IIFE setup?
It can potentially break execution